app/lib/features/news/widgets/news_item.dart

```diff
@@ -144,14 +108,80 @@ class NewsItem extends ConsumerWidget {
 
       default:
         return regularSlide(
+          context,
           Expanded(
             child: Center(
               child: Text('$slideType slides not yet supported'),
             ),
           ),
+          roomId,
+          space,
+          slide,
+          bgColor,
+          fgColor,
         );
     }
   }
+
+  Stack regularSlide(
```

Comment:
I'm not very fan of using excessive widget methods because its performance anti-pattern. If its possible to extract, do it as first priority, or use private ```Widget``` class as it cached and prevents unnecessary rebuilds.

Reply:
Fixed by https://github.com/acterglobal/a3/pull/855/commits/d151cfd6f0ce7dac183bf6c3a300b8a55e590666